test/e2e/testnet/defaults.go

```diff
@@ -7,4 +7,4 @@ var DefaultResources = Resources{
 	Volume:        "1Gi",
 }
 
-const TxsimVersion = "a92de72"
+const TxsimVersion = "pr-3541"
```

Comment:
This points to the latest version of this PR, once merged, we can use any version of main instead.